public/javascripts/ng-apps.js

```diff
@@ -1,17 +1,21 @@
-var springInitiative = angular.module('springInitiative', ['ngRoute']);
+var springInitiative = angular.module('springInitiativeApp', ['ui.router']);
 
-springInitiative.config(function($routeProvider) {
-  $routeProvider
-    .when('/student', {
-      templateUrl: 'views/student.html',
+springInitiative.config(function($stateProvider, $urlRouterProvider) {
+
+  $urlRouterProvider.otherwise('/login');
```

Comment:
this should just be `/`since currently it'd direct logged-in people to `/login`. There's a listener you can add that'll check to see if you're logged in and if not it'll redirect you to `\login` (see `layout` branch for example. it's in router.js there)

But for now, this is fine.